docs/MODULES.md

```diff
@@ -428,4 +509,86 @@ This module is used for changing the saturation of the image.
 where `options` is an object with the following property:
 * saturation : saturation for the new image (between 0 and 2; default 0)
 
+## Threshold 
+
+Thresholding is used to create binary images.
+
+## Usage
+
+```js
+  sequencer.loadImage('PATH')
+           .addSteps('saturation',options)
+           .run()
+```
+
+inputs : {
+    "threshold": {
+      "type": "select",
+      "desc": "Type of Thresholding",
+      "default": "Manual Thresholding",
+      "values": ["Manual Thresholding", "Automatic Thresholding"]
+    },
+    "input": {
+      "type": "range",
+      "desc": "Threshold Value",
+      "default": "120",
+      "min": "0",
+      "max": "255",
+      "step": "1"
+    }
+}
+
+
+
+## Tint 
+
+It adds color tint to an image
+
+## Usage
+
+```js
+  sequencer.loadImage('PATH')
+           .addSteps('Tint',options)
+           .run()
+```
+
+inputs : {
+      "color":{
+        "type": "String",
+        "desc": "RGB values separated by a space",
+        "default": "0 0 255"
+      },
+      "factor": {
+          "type": "range",
+          "desc": "Amount of tint",
+          "default": "0.5",
+          "min": "0",
+          "max": "1",
+          "step": "0.01"
+      }
+}
+
+
+
+## White Balance
+
+This Change the colour balance of the image by adjusting the colour temperature.
+
+## Usage
+
+```js
+  sequencer.loadImage('PATH')
+           .addSteps('White Balance',options)
```

Comment:
White Balance should be written as `white-balance` as it is written in [this modules.js](https://github.com/publiclab/image-sequencer/blob/main/src/Modules.js) file.